### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.9 es 4.52 (11 novembre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#320] [Esercizi M-Z]
![es52](https://github.com/user-attachments/assets/369fc61d-ac55-45aa-9359-23a9aafd7f3c)

**Comment by AglaiaNorza** [comment]
![es](https://github.com/user-attachments/assets/a15eb799-9d54-4b80-baf9-e758175a8df5)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/03979276-f744-4c52-a9c6-42917ec1bd5c)
